src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -200,7 +200,7 @@ private function isNfs(string $dir): bool
 
         if (null === $mounts) {
             $mounts = [];
-            if ('/' === \DIRECTORY_SEPARATOR && $files = @file('/proc/mounts')) {
+            if ('/' === \DIRECTORY_SEPARATOR && file_exists('/proc/mounts') && $files = @file('/proc/mounts')) {
```

Comment:
is_file?

Reply:
is_readable?

Reply:
Maybe, this need to be checked to be sure there it no warning.

I can't update the PR before January 3rd.

Reply:
no worries, I'm doing the change now